app/assets/javascripts/components/annotations/annotations_cell.ts

```diff
@@ -1,23 +1,17 @@
 import { customElement, property } from "lit/decorators.js";
 import { ShadowlessLitElement } from "components/meta/shadowless_lit_element";
 import { html, TemplateResult } from "lit";
-import {
-    createUserAnnotation,
-    getUserAnnotationsByLine,
-    UserAnnotationData,
-    UserAnnotationFormData
-} from "state/UserAnnotations";
-import { getEvaluationId } from "state/Evaluations";
-import { isQuestionMode, isAnnotationVisible } from "state/Annotations";
-import { getSubmissionId } from "state/Submissions";
-import { getMachineAnnotationsByLine, MachineAnnotationData } from "state/MachineAnnotations";
+import { UserAnnotationData, UserAnnotationFormData, userAnnotationState } from "state/UserAnnotations";
```

Comment:
```suggestion
import { UserAnnotationFormData, userAnnotationState } from "state/UserAnnotations";
```

`UserAnnotationData` seems unused.